docs/data-sources/td_security_policies.md

```diff
@@ -0,0 +1,112 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_td_security_policies Data Source - terraform-provider-bloxone"
+subcategory: ""
```

Comment:
Please ensure subcategory is "Threat Defense". You might have to modify the template here. https://github.com/infobloxopen/terraform-provider-bloxone/blob/master/templates/resources.md.tmpl

Reply:
I was planning to do this in a separate PR , once this is merged ,  as the doc file for all TD objects i.e. access codes , pop regions , Domain Lists. etc. would be changed.